eZ/Bundle/EzPublishCoreBundle/Imagine/Cache/AliasGeneratorDecorator.php

```diff
@@ -103,4 +104,16 @@ private function getCacheKey(Field $field, VersionInfo $versionInfo, $variationN
             $variationName
         );
     }
+
+    private function getTagsForVariation(Field $field, VersionInfo $versionInfo, string $variationName): array
+    {
+        return [
+            'image-variation',
+            'image-variation-name-' . $variationName,
+            'image-variation-siteaccess-' . ($this->siteAccess ? $this->siteAccess->name : 'default'),
+            'image-variation-content-' . $versionInfo->getContentInfo()->id,
+            'image-variation-content-' . $versionInfo->getContentInfo()->id . '-version-' . $versionInfo->versionNo,
```

Comment:
Shouldn't these two use the already documented tags for content and content version?

It will mean they get cleared on publish, and version updates _(on matching version tag)_, but that wouldn't be so bad.

Reply:
They can, but I specifically wanted to be able to clear variation cache for content and for a version, without republishing the content first.

Come to think of it, we can have those two you mentioned, and only image variation by content from this patch, since clearing variation cache for previous versions is not that useful.

Reply:
That would specifically result in what tags to add here?

Reply:
It would result to this:

```
'image-variation',
'image-variation-name-' . $variationName,
'image-variation-siteaccess-' . ($this->siteAccess ? $this->siteAccess->name : 'default'),
'image-variation-content-' . $versionInfo->getContentInfo()->id,
'content-' . $versionInfo->getContentInfo()->id,
'content-' . $versionInfo->getContentInfo()->id . '-version-' . $versionInfo->versionNo,
```

Reply:
ok from my side, but you skipped `'image-variation-field-' . $field->id,` on purpose?

Reply:
Whoops, no, it was below the comment so I forgot it when copy pasting :) Updated my previous comment.

Reply:
Go for it :)